app/models/spree/app_configuration.rb

```diff
@@ -149,5 +149,8 @@ class AppConfiguration < Preferences::Configuration
 
     # Available units
     preference :available_units, :string, default: "g,kg,T,mL,L,kL"
+
+    # Enable DFC API
+    preference :enable_dfc_api?, :boolean, default: false
```

Comment:
This is also something important I put on this MR, I added a new preference in order to activate or not the DFC API. This will be much secure for instances who don't want to use the feature.